### ISSUE TITLE
No "most" modules in 2.2-alpine 

### ISSUE THREAD

**Issue by ankon** [docker-library/httpd#36]
#6 / #8 fixed the issue for the regular 2.2 image, but it would be nice if the same fix would be applied to the alpine version as well.

**Comment by ankon** [comment]
FWIW:
~~~~
$ docker run --rm -it --name test httpd:2.4-alpine ls modules
httpd.exp                    mod_info.so
mod_access_compat.so         mod_isapi.so
mod_actions.so               mod_lbmethod_bybusyness.so
mod_alias.so                 mod_lbmethod_byrequests.so
mod_allowmethods.so          mod_lbmethod_bytraffic.so
mod_asis.so                  mod_lbmethod_heartbeat.so
mod_auth_basic.so            mod_ldap.so
mod_auth_digest.so           mod_log_config.so
mod_auth_form.so             mod_log_debug.so
mod_authn_anon.so            mod_log_forensic.so
mod_authn_core.so            mod_logio.so
mod_authn_dbd.so             mod_macro.so
mod_authn_dbm.so             mod_mime.so
mod_authn_file.so            mod_mime_magic.so
mod_authn_socache.so         mod_negotiation.so
mod_authnz_fcgi.so           mod_optional_fn_export.so
mod_authnz_ldap.so           mod_optional_fn_import.so
mod_authz_core.so            mod_optional_hook_export.so
mod_authz_dbd.so             mod_optional_hook_import.so
mod_authz_dbm.so             mod_proxy.so
mod_authz_groupfile.so       mod_proxy_ajp.so
mod_authz_host.so            mod_proxy_balancer.so
mod_authz_owner.so           mod_proxy_connect.so
mod_authz_user.so            mod_proxy_express.so
mod_autoindex.so             mod_proxy_fcgi.so
mod_bucketeer.so             mod_proxy_ftp.so
mod_buffer.so                mod_proxy_hcheck.so
mod_cache.so                 mod_proxy_http.so
mod_cache_disk.so            mod_proxy_scgi.so
mod_cache_socache.so         mod_proxy_wstunnel.so
mod_case_filter.so           mod_ratelimit.so
mod_case_filter_in.so        mod_reflector.so
mod_cern_meta.so             mod_remoteip.so
mod_cgi.so                   mod_reqtimeout.so
mod_cgid.so                  mod_request.so
mod_charset_lite.so          mod_rewrite.so
mod_data.so                  mod_sed.so
mod_dav.so                   mod_session.so
mod_dav_fs.so                mod_session_cookie.so
mod_dav_lock.so              mod_session_dbd.so
mod_dbd.so                   mod_setenvif.so
mod_deflate.so               mod_slotmem_plain.so
mod_dialup.so                mod_slotmem_shm.so
mod_dir.so                   mod_socache_dbm.so
mod_dumpio.so                mod_socache_memcache.so
mod_echo.so                  mod_socache_shmcb.so
mod_env.so                   mod_speling.so
mod_example_hooks.so         mod_ssl.so
mod_example_ipc.so           mod_status.so
mod_expires.so               mod_substitute.so
mod_ext_filter.so            mod_suexec.so
mod_file_cache.so            mod_unique_id.so
mod_filter.so                mod_unixd.so
mod_headers.so               mod_userdir.so
mod_heartbeat.so             mod_usertrack.so
mod_heartmonitor.so          mod_version.so
mod_ident.so                 mod_vhost_alias.so
mod_imagemap.so              mod_watchdog.so
mod_include.so
~~~~

~~~~
$ docker run --rm -it --name test httpd:2.2-alpine ls modules
httpd.exp
~~~~

**Comment by yosifkit** [comment]
This was fixed in #35 and pushed up today with https://github.com/docker-library/official-images/pull/2398.  Let us know if you still have any issues with the `2.2-alpine` image.

**Comment by ankon** [comment]
Tested this image, and the issue seems to be fixed.

Thanks a lot!
